core/dbt/graph/selector.py

```diff
@@ -31,6 +31,8 @@ def can_select_indirectly(node):
     """
     if node.resource_type == NodeType.Test:
         return True
+    elif node.resource_type == NodeType.Unit:
+        return True
```

Comment:
What are the implications of this? If I do a `dbt run --select model` on `model` that has unit tests defined, do the unit tests actually get run by the run task? Or is it a no-op because the run manifest does not contain `UnitTestNode`s? Either way, I think we should leave unit test execution out of the `dbt run` functionality since that command is typically used in production jobs in addition to development ones.